### PR TITLE
Migrate old PrismaScans to DemonSect

### PR DESCRIPTION
**Pull request by julioreigen**
**Info:**
PrismaScans changed their scan name and website url, so I updated the aidoku source to work correctly.
I put source version in 1, since it's like a new source.
New URL: [demonsect.com.br](https://demonsect.com.br)

**Checklist:**
- [x] Updated source's version for individual source changes
- [x] Set appropriate `nsfw` value
- [x] Did not change `id` even if a source's name or language were changed
- [x] Tested the modifications by running it on the simulator or a test device

### CODE REVIEW

File: src/rust/madara/sources/demonsect/res/source.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+			"name": "Latest"
+		},
+		{
+			"name": "Trending"
+		},
+		{
+			"name": "Popular"
+		}
+	]
+}
```
Comment (Skittyblock) [inline src/rust/madara/sources/demonsect/res/source.json:21]: Make sure there are newlines at the end of files
```suggestion
}

```

**Review by Skittyblock — CHANGES_REQUESTED** [review]

**Comment by Skittyblock** [comment]
Source doesn't seem work for me so I can't to test it

**Comment by julioreigen** [comment]
> Source doesn't seem to work for me so I can't test it

So I'm going to close this PR and make two others: One to delete PrismaScan and another to add DemonSect source.

**Comment by Skittyblock** [comment]
Uhh I don't think making two different PRs will fix anything. You can just make your changes here on this PR.

**Comment by julioreigen** [comment]
@Skittyblock I checked the font's website and it looks like it went into maintenance on the 23rd of this month and is redirecting to discord, which is why the source isn't working now.
I will let you know when the site is back online so you can check the source.

**Comment by dodieboy** [comment]
I believe it is back online

**Comment by julioreigen** [comment]
Yeah, it's back and all the manga have only the last chapters released, it's not a source problem.

**Comment by julioreigen** [comment]
@Skittyblock I updated the source and now it's fully working!
Tested on my iphone.

**Review by Skittyblock — APPROVED** [review]
I personally still can't access the site, but the code looks fine and if it works for you then it's probably good to go.

**Comment by julioreigen** [comment]
> I personally still can't access the site, but the code looks fine and if it works for you then it's probably good to go.

Probably they are denying access from IPs outside brazil.

**Pull request merged**